scss/_variables.scss

```diff
@@ -6,10 +6,14 @@ $serif: georgia, serif !default;
 $code: consolas, monaco, monospace !default;
 $font-size-headline: 6rem !default;
 $font-size-subheadline: 5rem !default;
+$font-size-0: 4rem !default;
 $font-size-1: 3rem !default;
+$font-size-15: 2.5rem !default;
 $font-size-2: 2.25rem !default;
```

Comment:
how about we stick to the in between classes being exactly halfway between?

Reply:
Do you mean going forward, or for the f15 class? It's already in use in the application so I'd want to see how it affects the look or move current uses of the f15 class to css.

Reply:
to clarify, I'm copying our prexisting tachyons-style classes into this repo, not inventing new ones

Reply:
i'm ok with porting it over as is in the app; we can review this separately to avoid regressions?